containers/elabctl/elabctl.sh

```
@@ -0,0 +1,687 @@
+            get-user-conf
+            exit 0
+        fi
+    fi
+
+    # create the data dir
+    mkdir -pv $DATA_DIR
+    if [ $? -eq 1 ]; then
+        sudo mkdir -pv $DATA_DIR
+    fi
+
+    declare TMP_DIR=$(mktemp -d)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)